README.md

```diff
@@ -720,23 +720,25 @@ interface Table extends Parent {
 ### CustomCodeComponent
 
 ```ts
-interface CustomCodeComponent extends Parent {
+type Attributes = {
```

Comment:
**question:** is the intention that any node that might have attributes would follow this type? If it's CCC specific, it might be better to name it `CustomCodeComponentAttributes` (or even leave it inline?)

Reply:
It's ccc specific, I've updated 🙏